src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -0,0 +1,165 @@
+        category: "Layers",
+        description: "Load ArcGIS vector tiled layers using custom styles.",
+        instructions: "Pan and zoom to explore the vector tile basemap.",
+        tags: new[] { "tiles", "vector", "vector basemap", "vector tiled layer", "vector tiles" })]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData("f4b742a57af344988b02227e2824ca5f")]
+    public partial class AddVectorTiledLayerFromCustomStyle
+    {
+        // ArcGIS Online portal item strings.
+        private readonly string[] _onlineItemIDs =
+        {
+            "1349bfa0ed08485d8a92c442a3850b06",
+            "bd8ac41667014d98b933e97713ba8377",
+            "02f85ec376084c508b9c8e5a311724fa",
+            "1bf0cc4a4380468fbbff107e100f65a5",
+        };
+
+        private readonly string[] _offlineItemIDs =
+        {
+            "e01262ef2a4f4d91897d9bbd3a9b1075",
+            "ce8a34e5d4ca4fa193a097511daa8855"
+        };
+
+        // Items to fill the ActionDisplaySheet for switching custom styles.
+        private readonly string[] styles = { "Default", "Style 1", "Style 2", "Style 3", "Offline custom style: Light", "Offline custom style: Dark" };
+
+        // Path to Dodge City vector tile package.
+        private readonly string _localVectorPackagePath = DataManager.GetDataFolder("f4b742a57af344988b02227e2824ca5f", "dodge_city.vtpk");
+
+        private List<PortalItem> _vectorTiledLayers = new List<PortalItem>();
+
+        private readonly Viewpoint _defaultViewpoint = new Viewpoint(10, 5.5, 1e8);
+        private readonly Viewpoint _dodgeCityViewpoint = new Viewpoint(37.76528, -100.01766, 4e4);
+
+        private ItemResourceCache _lightStyleResourceCache;
+        private ItemResourceCache _darkStyleResourceCache;
+
+        public AddVectorTiledLayerFromCustomStyle()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Load the default portal.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Store a list of all portal items.
+                foreach (string item in _onlineItemIDs)
+                {
+                    PortalItem portalItem = await PortalItem.CreateAsync(portal, item);
+                    _vectorTiledLayers.Add(portalItem);
+                }
+                foreach (string item in _offlineItemIDs)
+                {
+                    PortalItem portalItem = await PortalItem.CreateAsync(portal, item);
+                    _vectorTiledLayers.Add(portalItem);
+                }
+
+                // Create a map using defaults.
+                MyMapView.Map = new Map(new Basemap(new ArcGISVectorTiledLayer(_vectorTiledLayers[0]))) { InitialViewpoint = _defaultViewpoint };
+
+                // By default, the UI label will not reflect the default style.
+                ChosenStyle.Text = "Default";
```

Comment:
```suggestion
                ChosenStyle.Text = "Current style: Default";
```
I think this makes it clearer to the user what "Default" means on sample load.